### ISSUE TITLE
Getting error while deploying DocumentTemplate

### ISSUE THREAD

**Issue by irfanlai** [vlocityinc/vlocity_build#578]
We are using Github actions and gitcheck to deploy delta changes to the target org. While deploying DocumentTemplate, we are getting the following error. 

Any other way we can get more details on the error?
I don't, want to manually import the data pack, any solution for this issue?

```
Formatting Manifest  
  Git Hash >> 51d50b4fad3168605098ac674f239491d8496232  
  Running Job >> Deploy  
  Deploying Vlocity  
  Adding to Deploy >> DocumentTemplate/Fair_Credit_Reporting_Spanish - Fair_Credit_Reporting_Spanish    
  Deploy >> 1 Items  
  Deploying >> DocumentTemplate/Fair_Credit_Reporting_Spanish  
  Result >> {"Finished":0,"Total":1,"Message":"Ready","Status":"Ready","VlocityDataPackId":"a5W7j000000LCzSEAW","VlocityDataPackProcess":"Import"}  
  Result >> {"Finished":1,"Total":1,"Message":"Error","Status":"Error","VlocityDataPackId":"a5W7j000000LCzSEAW","VlocityDataPackProcess":"Import"}  
  Error >> DocumentTemplate/Fair_Credit_Reporting_Spanish -- DataPack >> Fair_Credit_Reporting_Spanish -- Error Message -- This DataPack has a reference to another object which was not found -- ContentVersion/df83f321-d7f6-57bb-8063-fd84af893d79  
  Changing >> No match found for vlocity_cmt__DocumentTemplate__c.vlocity_cmt__TemplateContentVersionId__c - vlocity_cmt__GlobalKey__c=df83f321-d7f6-57bb-8063-fd84af893d79 (System Code) >> DocumentTemplate/Fair_Credit_Reporting_Spanish -- DataPack >> Fair_Credit_Reporting_Spanish -- Error Message -- This DataPack has a reference to another object which was not found -- ContentVersion/df83f321-d7f6-57bb-8063-fd84af893d79  
  Result >> {"Status":"Complete","VlocityDataPackId":"a5W7j000000LCzSEAW","VlocityDataPackProcess":"Activate"}  
  Result >> {"Status":"Deleted","VlocityDataPackId":"a5W7j000000LCzSEAW"}  
  Error >> 1  
  Salesforce Org >> ***  
  Version Info >> v1.17.1 vlocity_cmt 900.452  
  Log File >> vlocity-temp/logs/buildfiles/deploy_to_qa-yaml-2023-03-17T14-45-34-325Z-Deploy.yaml  
  Current Status >> Deploy  
  Remaining >> 0  
  Success >> 0  
  Elapsed Time >> 0m 21s  
  Checking Valid Imports Start  
  Saving File Start  
  Saving File End >> 355  
  Retrying Deployment  
  Finalizing Deployment  
  Finalizing Deployment  
  Salesforce Org >> ***  
  Error >> 1  
  Version Info >> v1.17.1 vlocity_cmt 900.452  
  Log File >> vlocity-temp/logs/buildfiles/deploy_to_qa-yaml-2023-03-17T14-45-34-325Z-Deploy.yaml  
  Current Status >> Deploy  
  Remaining >> 0  
  Success >> 0  
  Elapsed Time >> 0m 22s  
  Formatting Response >> Deploy  
  DocumentTemplate/Fair_Credit_Reporting_Spanish >> Error  
  Saving File Start  
  Saving File End >> 333  
  Job Complete >> Deploy 0m 22s  
Deploy error:
DocumentTemplate/Fair_Credit_Reporting_Spanish -- DataPack >> Fair_Credit_Reporting_Spanish -- Error Message -- This DataPack has a reference to another object which was not found -- ContentVersion/df83f321-d7f6-57bb-8063-fd84af893d79
Error: Process completed with exit code 1.
```

**Comment by 92SGupta** [comment]
You need to deploy the content version(Actual doc) as well which would be referenced in the Document Templates. If you have already included the content Version then the issue will be resolved with PR #579.

**Comment by irfanlai** [comment]
We were able to download the dependencies and the issue is fixed now.
